modules/nf-core/sentieon/dnascope/meta.yml

```diff
@@ -0,0 +1,100 @@
+name: sentieon_dnascope
+description: DNAscope algorithm performs an improved version of Haplotype variant calling.
+keywords:
+  - dnascope
+  - sentieon
+  - variant_calling
+tools:
+  - sentieon:
+      description: |
+        Sentieon® provides complete solutions for secondary DNA/RNA analysis for a variety of sequencing platforms, including short and long reads.
+        Our software improves upon BWA, STAR, Minimap2, GATK, HaplotypeCaller, Mutect, and Mutect2 based pipelines and is deployable on any generic-CPU-based computing system.
+      homepage: https://www.sentieon.com/
+      documentation: https://www.sentieon.com/
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information.
+        e.g. [ id:'test', single_end:false ]
+  - meta2:
+      type: map
+      description: |
+        Groovy Map containing reference information.
+        e.g. [ id:'test' ]
+  - meta3:
+      type: map
+      description: |
+        Groovy Map containing reference information.
+        e.g. [ id:'test' ]
+  - meta4:
+      type: map
+      description: |
+        Groovy Map containing reference information.
+        e.g. [ id:'test' ]
+  - meta5:
+      type: map
+      description: |
+        Groovy Map containing reference information.
+        e.g. [ id:'test' ]
+  - meta6:
+      type: map
+      description: |
+        Groovy Map containing reference information.
+        e.g. [ id:'test' ]
+  - bam:
+      type: file
+      description: BAM file.
+      pattern: "*.bam"
+  - bai:
+      type: file
+      description: BAI file
+      pattern: "*.bai"
+  - fasta:
+      type: file
+      description: Genome fasta file
+      pattern: "*.{fa,fasta}"
+  - fai:
+      type: file
+      description: Index of the genome fasta file
+      pattern: "*.fai"
+  - dbsnp:
+      type: file
+      description: Single Nucleotide Polymorphism database (dbSNP) file
+      pattern: "*.vcf.gz"
+  - dbsnp_tbi:
+      type: file
+      description: Index of the Single Nucleotide Polymorphism database (dbSNP) file
+      pattern: "*.vcf.gz.tbi"
+  - call_interval:
+      type: file
+      description: bed or interval_list file containing interval in the reference that will be used in the analysis
+      pattern: "*.{bed,interval_list}"
+  - ml_model:
+      type: file
+      description: machine learning model file
+      pattern: "*.model"
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing reference information.
+        e.g. [ id:'test', single_end:false ]
+  - vcf:
+      type: file
+      description: VCF file
+      pattern: "*.{vcf.gz}"
+  - index:
+      type: file
+      description: Index of the VCF file
+      pattern: "*.crai"
+  - vcf_index:
```

Comment:
Depending on which changes are made in the output-section, this will also have to be changed.